myft/ui/save-article-to-list-variant.js

```diff
@@ -23,6 +23,7 @@ export default async function openSaveArticleToListVariant (name, contentId) {
 					overlayContent.insertAdjacentElement('afterbegin', listElement);
 					const announceListContainer = document.querySelector('.myft-ui-create-list-variant-announcement');
 					announceListContainer.textContent = `${list} created`;
+					triggerCreateListEvent(contentId)
```

Comment:
```suggestion
					triggerCreateListEvent(contentId);
```
Nitpick: because eslint was complaining 😁